### PR TITLE
[FEATURE-SERVERSIDE-APPLY] Mark the kubectl flag as alpha

### PR DESCRIPTION
**Pull request by apelisse**
**What type of PR is this?**
/kind feature
/sig api-machinery
/priority important-soon
/assign @jennybuckley @kwiesmueller 

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

**Comment by apelisse** [comment]
@soltysh

File: pkg/kubectl/cmd/util/helpers.go [hunk 1]
```diff
@@ -401,8 +401,8 @@ func AddDryRunFlag(cmd *cobra.Command) {
 }
 
 func AddServerSideApplyFlags(cmd *cobra.Command) {
-	cmd.Flags().Bool("server-side", false, "If true, apply runs in the server instead of the client.")
-	cmd.Flags().Bool("force-conflicts", false, "If true, server-side apply will force the changes against conflicts.")
+	cmd.Flags().Bool("server-side", false, "ALPHA -- If true, apply runs in the server instead of the client.")
```
Comment (soltysh) [inline pkg/kubectl/cmd/util/helpers.go]: Change this to `This is an alpha feature and flag` similarly to how we do it in https://github.com/kubernetes/kubernetes/blob/00eab3c40bcda9ee73eac060242d36602e627fec/pkg/kubectl/cmd/apply/apply.go#L179

**Review by soltysh — CHANGES_REQUESTED** [review]

**Comment by soltysh** [comment]
> @soltysh do you mind looking at this? Bear in mind that this is for the feature branch and we're trying to get the branch back in master by the end of the week, thanks

Yup, I'm about to look into it too :)

**Review by soltysh — APPROVED** [review]
/lgtm

**Pull request merged**